frontend/pages/index.vue

```diff
@@ -2,37 +2,25 @@
   <v-container>
     <v-row class="ma-0">
       <v-col cols="12" class="dashboard-grid">
-        <EquipmentCard
-          v-for="equipment in equipments.data"
-          :key="equipment.id"
-          :equipment="equipment"
-          @click="navigateTo(`/equipment/${equipment.id}`)"
-        />
+        <template v-if="equipments">
```

Comment:
Could we add a `v-else` to show the user if there aren't any equipment? Or if any are loading?